app/src/main/res/layout/topic_practice_fragment.xml

```diff
@@ -13,9 +13,12 @@
     android:id="@+id/topic_practice_skill_list"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/oppiaBackground"
     android:clipToPadding="false"
```

Comment:
@rt4914 only sw600dp files have background color as `oppiaBackground`. Is it something that mock demands or we missed to add background color to mobile layout files.

Reply:
We missed it. It should be in both the files.